juniper_warp/Cargo.toml

```diff
@@ -20,13 +20,13 @@ juniper_graphql_ws = { version = "0.2.2", path = "../juniper_graphql_ws", option
 serde = { version = "1.0.75", features = ["derive"] }
 serde_json = "1.0.24"
 thiserror = "1.0"
-tokio = { version = "0.2", features = ["blocking", "rt-core"] }
+tokio = { version = "1.0.2", features = ["rt"] }
 warp = "0.2"
```

Comment:
Should this be updated to warp version `0.3` (which is warp upgrading to Tokio `1.0`) as well? 😅

Reply:
Yep, seems so. Gotta check the other integrations as well. 

For now I stumbled upon actix integrations test. 

Also I have tests that pass locally yet fail on CI :(

Reply:
I've done some quick and dirty upgrading to get warp `0.3` working [on my fork](https://github.com/TilBlechschmidt/juniper/commit/603f588249c61b551f0f8c9874f06b8876164922) (which is a fork of @akhramov 's fork). 

Apart from bumping the version, it also required a change from `bytes::Bytes` to `warp::hyper::body::Bytes` in `lib.rs` and the runtime used for the example has to be set to `rt-multi-thread` since that appears to be the default now for the `tokio::main` macro.

Hope it helps :)